src/Data/Textile/Process.elm

```diff
@@ -35,6 +35,9 @@ type alias Process =
     { name : String
     , info : String
     , unit : String
+    , source : String
+    , correctif : String
+    , step_usage : String
```

Comment:
Nit: I know `elec_pppm` is in snake_case but it's currently one of the few exceptions to the general rule of using cameCase for record property names. FTR, I don't have a strong preference between these two styles but I'd really love consistency across the codebase.

Reply:
Sure, renaming now 👍